### PR TITLE
Bag

### PR DESCRIPTION
**Pull request by AshutoshM29**
PR for the cart page
Preview: https://preview-urban-tails.netlify.app/pages/bag

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
looks good

File: Pages/bag.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    padding: 1rem 0.5rem;
+    text-align: center;
+    margin: 5rem 0.5rem;
+}
+
+.value {
+    color: var(--gray-mild);
+    border: 0.2rem solid var(--gray-lighter);
+    border-radius: 0.5rem;
+    padding: 0 0.5rem;
```
Comment (Rishabh-078) [inline Pages/bag.css:99]: It was suggested to use px then the value is less than 1rem.

**Comment by Rishabh-078** [comment]
Everything is looking good, can be merged

**Review by Sahil25061999 — APPROVED** [review]
suggestion : can you make the quantity button of different color gray generally indicates disabled, but its not necessary
Everything else looks good

**Comment by AshutoshM29** [comment]
> suggestion : can you make the quantity button of different color gray generally indicates disabled, but its not necessary Everything else looks good

ok, thanks for the reply

**Pull request merged**